src/home/github-types.ts

```diff
@@ -123,6 +123,10 @@ export interface AvatarCache {
 
 export const GITHUB_TASKS_STORAGE_KEY = "gitHubTasks";
 
+// supabase key should be dynamic incase of change and testing
+const supabaseUrl = process.env.SUPABASE_URL ? process.env.SUPABASE_URL.split(".")[0] : "";
```

Comment:
You need to check that this value exists and throw an error if it doesn't before trying to use it.

Reply:
It's already throwing an error before it even gets called here, but I'd add a check here too